src/components/formattedNumber/index.js

```diff
@@ -1,20 +1,28 @@
 import React from 'react';
+import BigNumber from 'bignumber.js';
+
+/**
+ *
+ * @param {*} num - it is a number that we want to normalize it
+ * @return {string} - normalized version of input number
+ */
+const normalize = value => new BigNumber(value || 0).dividedBy(new BigNumber(10).pow(8)).toFixed();
 /**
  *
  * @param {*} num - it is a number that we want to format it as formatted number
  * @return {string} - formatted version of input number
  */
 const formatNumber = (num) => {
-  num = parseFloat(num);
-  const sign = num < 0 ? '-' : '';
-  const absVal = String(parseInt(num = Math.abs(Number(num) || 0), 10));
+  let normalizeNum = parseFloat(normalize(num));
```

Comment:
`normalize(num)` IMO should not be in `<FormattedNumber />` component
I'm using `<FormattedNumber />` to format "Block height" in "Forged blocks", which is a number, but not LSK value. 
I think we need another component `<LiskAmount val={val} />` that will render something like `LiskAmount =
 props => (<FormattedNumber val({normalize(props.val)}/> LSK)`. 
But I will create a separate issue for it. This is already big enough.